transcom/mymove
# B 19035 update payment reminder email template

## B-19035

## Summary

Integration PR: https://github.com/transcom/mymove/pull/12204

As a Customer  you should receive an email if you haven't submitted any docs within the last 14 days of today.
This PR updates the original template and updates the trigger.
Also adds back in the send-payment-reminder task to main.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Create a move with an estimateDepartureDate that is >= 14 days from today(i.e. today = January 14, 2024 estimateDeparture  could be January 1, 2024 ), and has no null estimate values(incentive, weight), and set the personal email to one you can access
2. Approve this move as a service counselor
3. Run `aws-vault exec transcom-gov-dev` to get into the subshell
4. Run `go run ./cmd/milmove-tasks send-payment-reminder --email-backend=ses --aws-ses-domain=devlocal.dp3.us  --aws-ses-region=us-gov-west-1`
5. You should get a payment reminder email in a couple minutes max

## Screenshots
**Actual Email**
<img width="1728" alt="Screenshot 2024-03-14 at 10 40 41 AM" src="https://github.com/transcom/mymove/assets/146969726/d628fe90-de31-47ac-a78e-0bb3f5c7b449">


**Email template from PO**
[Complete your PPM EMail_Final.docx](https://github.com/transcom/mymove/files/14604668/Complete.your.PPM.EMail_Final.docx)
